native/dggsvp3.go

```diff
@@ -260,7 +260,7 @@ func (impl Implementation) Dggsvp3(jobU, jobV, jobQ lapack.GSVDJob, m, p, n int,
 		}
 
 		// Clean up A.
-		for i := k + 1; i < k+l; i++ {
+		for i := k + 1; i < min(m, k+l); i++ {
```

Comment:
I don't think it really matters (I don't think the elements of the block at A[k+l:m, n-l:n] are referenced later, but I will change to make sure they are zeroed.